server/polar/models/discount.py

```diff
@@ -57,7 +57,24 @@ class DiscountDuration(StrEnum):
 
 class Discount(MetadataMixin, RecordModel):
     __tablename__ = "discounts"
-    __table_args__ = (UniqueConstraint("organization_id", "code"),)
+
+    @declared_attr
+    def __table_args__(cls):
```

Comment:
I think mypy complains because of missing type annotations here. Two things:

* SQLAlchemy recommends to use `@declared_attr.directive` to please type checkers: https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.declared_attr.directive
* You should explicitly set a return type; say, `Sequence[Index]` for ease

---

You can run the type checker like this:

```
uv run task lint_types
```

Or all in one:

```
uv run task lint && uv run task lint_types
```